Bender.yml

```diff
@@ -166,6 +165,8 @@ sources:
       - hw/snitch_cluster/src/snitch_fp_ss.sv
       - hw/snitch_cluster/src/snitch_shared_muldiv.sv
       - hw/snitch_cluster/src/snitch_tcdm_interconnect.sv
+      - hw/snitch_cluster/src/snitch_fpu.sv
+      - hw/snitch_cluster/src/snitch_ipu.sv
       # Level 2
```

Comment:
What level 0 files in the snitch_cluster group do these depend on? I don't know if this is correct, but if so then e.g. snitch_fp_ss would have to become level 2.

Also within each level files are listed in alphabetical order.